NightscoutServiceKit/RemoteCommands/V1/RemoteCommandSourceV1.swift

```diff
@@ -14,19 +14,27 @@ class RemoteCommandSourceV1: RemoteCommandSource {
     weak var delegate: RemoteCommandSourceV1Delegate?
     private let otpManager: OTPManager
     private let log = OSLog(category: "Remote Command Source V1")
+    private var commandValidator: RemoteCommandValidator
+    private var recentNotifications = RecentNotifications()
     
     init(otpManager: OTPManager) {
         self.otpManager = otpManager
+        self.commandValidator = RemoteCommandValidator(otpManager: otpManager)
     }
     
-    
     //MARK: RemoteCommandSource
     
     func remoteNotificationWasReceived(_ notification: [String: AnyObject]) async {
+        
         do {
             guard let delegate = delegate else {return}
             let remoteNotification = try notification.toRemoteNotification()
```

Comment:
This is the bug fix. We track recent notifications in memory. If the notification was already received, we silently bail out.